### ISSUE TITLE
`Set<ChunkPos>` Component (e.g. Claims)

### ISSUE THREAD

**Issue by YOUNIN15** [sleepingdragoninn/surveyor#47]
Open Parties and Claims is one of the most popular mods in chunks claims, I don't know if it is possible to add compatibility with this mod since it has no integrated map and depends only on external maps.

I reported this to the developers of Open Parties and Claims and Antique Atlas 4.

Open Parties and Claims: I was told that they cannot add support, they must be added by the map developers themselves.

Antique Atlas 4: I was told that suveryor would have to add support first.

**Comment by sleepingdragoninn** [comment]
Yep, surveyor was built with the potential for support for generic claim displays in mind - the actual compat will be done in an addon (like how [Surveystones](https://modrinth.com/mod/surveystones) works) but we'll need to add generic "chunk area" landmarks in surveyor first.

**Comment by YOUNIN15** [comment]
> Yep, surveyor was built with the potential for support for generic claim displays in mind - the actual compat will be done in an addon (like how [Surveystones](https://modrinth.com/mod/surveystones) works) but we'll need to add generic "chunk area" landmarks in surveyor first.

Hopefully soon compatibility can be added, a good idea would be that instead of showing an area as such, it could be that where there is a claim, some kind of special icon appears with the name of the player who has the claim.

**Comment by YumYumJudah** [comment]
@sisby-folk Do you know when you plan to work on this, I'm creating a modpack/server with both Surveyor/OPAC which will be out mid december and this would be very helpful. I'd be happy to donate/support to speed up production if at all possible

**Comment by eatenpancreas** [comment]
Is there any update on this? A "claimed chunk" landmark would be a great starting point, i'd love to try making a compat mod

**Comment by sleepingdragoninn** [comment]
1.0 is in alpha right now but it throws out the need for base types in surveyor itself now that we're using components. will add a `Set<ChunkPos>` component before 1.0 full release, complain if we don't.

**Comment by sleepingdragoninn** [comment]
claim support is now fully implemented in the alpha ecosystem (incl hoofprint, aa4) https://github.com/sisby-folk/surveyalot
